build/build.clj

```diff
@@ -40,7 +40,8 @@
                         (worktree-dirty?)
                         (conj "DIRTY"))
                       (string/join "-"))]
-    (format "%s/%s.jar" target filename)))
+    #_(format "%s/%s.jar" target filename)
```

Comment:
we'll need to handle this better with a flag to give the uberjar an assigned name, so that I can build it with a fixed name and copy it out.